components/behaviors/time-series.schema.json

```diff
@@ -44,6 +44,8 @@
             "advertising.thirdQuartiles": "Advertising Third Quartiles",
             "web.webpagedetails.pageViews": "Web Webpagedetails Page Views",
             "web.webinteraction.linkClicks": "Web Webinteraction Link Clicks",
+            "web.fillOutForm": "Web Fill Out Form",
```

Comment:
What is the process if PM (or people) wants to change the name, such as "Web Fill Out Form", for any of the new ones?